models/ht_neuron.cpp

```diff
@@ -49,8 +49,7 @@ RecordablesMap< ht_neuron >::create()
     Name( "Theta" ), &ht_neuron::get_y_elem_< ht_neuron::State_::THETA > );
   insert_(
     Name( "g_AMPA" ), &ht_neuron::get_y_elem_< ht_neuron::State_::G_AMPA > );
-  insert_(
-    Name( "g_NMDA" ), &ht_neuron::get_y_elem_< ht_neuron::State_::G_NMDA > );
+  insert_( Name( "g_NMDA" ), &ht_neuron::get_g_NMDA_ );
```

Comment:
Naming is a bit of a mess here. In the paper, it is \tilde{g}_NMDA = m(V) g_NMDA(t), where g_NMDA(t) is the double exponential function.

I think that the end user will be most interested in the actual conductance of NMDA channels when Mg-blocking is taken into account (the m(V)). Therefore, I think the recordable should be `g_NMDA` and the getter `get_g_NMDA_()`, but I think I will change the name of the state variable, maybe to `G_NMDA_TIMECOURSE`.